core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -366,7 +431,7 @@ class AdminClientIntegrationTest extends KafkaServerTestHarness with Logging {
 
   override def generateConfigs = {
     val cfgs = TestUtils.createBrokerConfigs(brokerCount, zkConnect, interBrokerSecurityProtocol = Some(securityProtocol),
-      trustStoreFile = trustStoreFile, saslProperties = serverSaslProperties)
+      trustStoreFile = trustStoreFile, saslProperties = serverSaslProperties, logDirCount = 2)
```

Comment:
Not sure if I fully understand your question. Do you mean something like `protected def logDirCount: Int = 1` which is added by this patch in `BaseRequestTest.scala`? I think we only need to add this when we need to vary the logDirCount based on the test. It seems OK and simpler to just set logDirCount to 2 for all tests in `AdminClientIntegrationTest`.